commodore/cluster.py

```diff
@@ -43,14 +43,16 @@ def _full_target(cluster, components, catalog):
     customer = cluster['tenant']
     component_defaults = [f"defaults.{cn}" for cn in components if
                           (P('inventory/classes/defaults') / f"{cn}.yml").is_file()]
+    global_defaults = ['global.common', f"global.{cluster_distro}", f"global.{cloud_provider}"]
+    if not cluster_distro:
+        raise click.ClickException(f"Required fact 'distribution' not set")
+    if not cloud_provider:
+        raise click.ClickException(f"Required fact 'cloud' not set")
+    if cloud_region:
+        global_defaults.append(f"global.{cloud_provider}.{cloud_region}")
+    global_defaults.append(f"{customer}.{cluster_id}")
```

Comment:
There don't seem to be tests to ensure those changes. Please, don't forget to add them. This accumulates technical debt.